src/Symfony/Bundle/WebProfilerBundle/Resources/views/Icon/twig2.svg

```diff
@@ -0,0 +1,3 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="24" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
+    <path fill="#AAAAAA" d="m8.932327,22.491763c0.015209,-6.448229 -0.971579,-11.295129 -5.994828,-11.618392c4.689807,-0.35251 7.112354,2.633007 9.29714,6.906552c-0.030468,-11.426311 -2.352459,-16.227324 -7.433672,-16.482536c7.433365,0.069108 10.027703,5.898928 11.508208,14.292954c1.170879,-2.282766 3.560547,-5.553431 5.347074,-1.361535c-1.59464,-2.040144 -3.607126,-1.61697 -3.978662,8.261983l-8.744646,0.000107z"/>
```

Comment:
I added a new Twig icon (logo), maybe it could be a good opportunity to use it as the main icon of this panel?

| Before | After |
| --- | --- |
| ![twig-toolbar-icon-before](https://user-images.githubusercontent.com/2028198/42820333-e35684c0-89a3-11e8-8b3a-f02bb835103c.png) | ![twig-toolbar-icon-after](https://user-images.githubusercontent.com/2028198/42820342-eac3a95e-89a3-11e8-929e-ef4fb1e392bb.png) 
| ![twig-profiler-icon-before](https://user-images.githubusercontent.com/2028198/42820356-f3f04b72-89a3-11e8-9209-41ed37cf0915.png) | ![twig-profiler-icon-after](https://user-images.githubusercontent.com/2028198/42820370-fef24336-89a3-11e8-9b09-5ba1bf97df67.png) |

What do you think?

Reply:
Updating the icon sounds good to me.

Reply:
Updated.